shiny/render/transformer/_transformer.py

```diff
@@ -551,20 +609,20 @@ def output_transformer(
     function, and any keyword arguments supplied to the output tranformer decorator:
 
     * The first parameter to the handler function has the class
-      :class:`~shiny.render.transformer.TransformerMetadata` and is typically called
-      `_meta`. This information gives context the to the handler while trying to
-      resolve the app-supplied value function (typically called `_fn`).
+    :class:`~shiny.render.transformer.TransformerMetadata` and is typically called
+    `_meta`. This information gives context the to the handler while trying to
+    resolve the app-supplied value function (typically called `_fn`).
```

Comment:
Just curious: did you do this, or did the text editor do it automatically?

Reply:
Oh, text editor. I didn't even realize.

Reply:
Weird, mine doesn't do that when I press Alt-Q. It leaves the indentation however it was before (either in line with the `*` or indented two spaces).

Reply:
Mine too. No idea how that happened then

Reply:
Fixed